lib/project_types/script/forms/script_form.rb

```diff
@@ -6,7 +6,7 @@ class ScriptForm < ShopifyCli::Form
       protected
 
       def organization
-        @organization ||= ask_organization(ctx)
+        @organization ||= ask_organization
```

Comment:
Not entirely related to this PR but when I do a `shopify logout` to log out of my partner account and then `shopify enable` to enable the script, I get stuck on line 14. Do you also get this behaviour or is it just me? 😅

Reply:
Yeah theres a bug there that I'm looking into now. Thanks for bringing that up!